### ISSUE TITLE
Share button: aria-hidden prevents it from being perceivable to a screen reader

### ISSUE THREAD

**Issue by jhung** [platform-coop-toolkit/coop-library#189]
**Describe the bug**
The Share button has a `aria-hidden=true` property which prevents it from being accessible to a screen reader.

**To Reproduce**
1. Start a screen reader like NVDA.
2. Use the screen reader and navigate / read through the page (not tabbing through)
3. Notice that the screen reader will read back the name of the button (i.e. "button collapsed Share") but can not focus and activate it.

**Expected behavior**
The share button should be accessible and operable through a screen reader. Perhaps implemented similar to the Language menu.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Firefox 73, NVDA

**Comment by greatislander** [comment]
Turns out this isn't due to `aria-hidden` (which is only on the SVG icon). We are still investigating; behaviour is not reproducible with VoiceOver / Safari.

**Comment by greatislander** [comment]
Closed in favour of https://github.com/platform-coop-toolkit/pinecone/issues/249
